vmdb/app/presenters/menu/default_menu.rb

```diff
@@ -0,0 +1,111 @@
+        Menu::Section.new(:inf, "Infrastructure", [
+          Menu::Item.new('ems_infra',        _('Providers'),        'ems_infra',     {:feature => 'ems_infra_show_list'},     '/ems_infra'),
+          Menu::Item.new('ems_cluster',      _('Clusters'),         'ems_cluster',   {:feature => 'ems_cluster_show_list'},   '/ems_cluster'),
+          Menu::Item.new('host',             _('Hosts'),            'host',          {:feature => 'host_show_list'},          '/host'),
+          Menu::Item.new('vm_infra',         _('Virtual Machines'), 'vm_infra_explorer_accords',
+                                                                                     {:feature => 'vm_infra_explorer_accords', :any => true},
+                                                                                                                              '/vm_infra/explorer'),
+          Menu::Item.new('resource_pool',    _('Resource Pools'),   'resource_pool', {:feature => 'resource_pool_show_list'}, '/resource_pool'),
+          Menu::Item.new('storage',          ui_lookup(:tables => 'storages'),
+                                                                    'storage',       {:feature => 'storage_show_list'},       '/storage'),
+          Menu::Item.new('repository',       _('Repositories'),     'repository',    {:feature => 'repository_show_list'},    '/repository'),
+          Menu::Item.new('pxe',              _('PXE'),              'pxe',           {:feature => 'pxe', :any => true},       '/pxe/explorer'),
+          Menu::Item.new('miq_request_host', _('Requests'),         nil,             {:feature => 'miq_request_show_list'},   '/miq_request?typ=host')
+        ])
+      end
+
+      def storage_menu_section
+        Menu::Section.new(:sto, "Storage", [
+          Menu::Item.new('ontap_storage_system', ui_lookup(:tables => 'ontap_storage_system'), 'ontap_storage_system', {:feature => 'ontap_storage_system_show_list'}, '/ontap_storage_system'),
+          Menu::Item.new('ontap_logical_disk',   ui_lookup(:tables => 'ontap_logical_disk'),   'ontap_logical_disk',   {:feature => 'ontap_logical_disk_show_list'},   '/ontap_logical_disk'),
+          Menu::Item.new('ontap_storage_volume', ui_lookup(:tables => 'ontap_storage_volume'), 'ontap_storage_volume', {:feature => 'ontap_storage_volume_show_list'}, '/ontap_storage_volume'),
+          Menu::Item.new('ontap_file_share',     ui_lookup(:tables => 'ontap_file_share'),     'ontap_file_share',     {:feature => 'ontap_file_share_show_list'},     '/ontap_file_share'),
+          # Menu::Item.new('cim_base_storage_extent', _('Base Extents'),                    'cim_base_storage_extent', {:feature => 'cim_base_storage_extent_show_list'}, '/cim_base_storage_extent'), # -if false
+          Menu::Item.new('storage_manager',      _('Storage Managers'),                        'storage_manager',      {:feature => 'storage_manager_show_list'}, '/storage_manager')
+          # FIXME: removed feature snia_local_file_system
```

Comment:
@dclarizio : I am unsure about this: `snia_local_file_system` I removed that from the list